src/distilabel/utils/logging.py

```diff
@@ -93,7 +93,7 @@ def setup_logging(
         )
         log_level = "INFO"
 
-    root_logger = logging.getLogger()
+    root_logger = logging.getLogger("distilabel.root")
```

Comment:
@My-laniaKeA - should this not be "distilabel" to have the parent level and not a sibling?